uwmmss0347.xml

```diff
@@ -304,6 +304,518 @@
 						</did>
 						<c04>
 							<did>
+								<unittitle><title render="doublequote">Bambi</title>,
+									<unitdate>17 June 1991</unitdate></unittitle>
+								<container type="Box" label="Box">5</container>
+								<physdesc><extent>Audio cassette 1</extent></physdesc>
+							</did>
+							<scopecontent>
+								<p>With Tom Clark, Political Correspondence</p>
```

Comment:
Is this the same Tom Clark as below? If so, I'd like to see his WPR credentials mentioned here as well